projects/npm-tools/packages/npm-scripts/support/generateTimingsReport.js

```diff
@@ -64,12 +63,13 @@ function main() {
 			const label = fields[0];
 			const delta = fields[3];
 
-			const ms = parseInt(delta);
+			const ms = parseInt(delta, 10);
 
-			(labels[label] = (labels[label] || 0) + ms),
-				(modules[file] = modules[file] || {
-					'~total': 0,
-				});
```

Comment:
I swear I didn't write it like this... and it doesn't look like the sort of thing Prettier would do... weird...

Reply:
Shame on you